java/java-jersey-jaxrs/src/main/java/io/swagger/sample/data/StoreData.java

```diff
@@ -18,12 +18,10 @@
 
 import io.swagger.sample.model.Order;
 
-import java.util.Date;
-import java.util.List;
-import java.util.ArrayList;
+import java.util.*;
 
 public class StoreData {
-  static List<Order> orders = new ArrayList<Order>();
+  static List<Order> orders = Collections.synchronizedList(new ArrayList<Order>());
```

Comment:
If you always access this in a synchronized block, there is no need for the additional synchronizedList.